tests/gamma/affinity_test.py

```diff
@@ -27,13 +29,22 @@
 logger = logging.getLogger(__name__)
 flags.adopt_module_key_flags(xds_k8s_testcase)
 
+# Type aliases.
+_Lang: TypeAlias = skips.Lang
 _XdsTestServer = server_app.XdsTestServer
 _XdsTestClient = client_app.XdsTestClient
 
 _REPLICA_COUNT = 3
 
 
 class AffinityTest(xds_gamma_testcase.GammaXdsKubernetesTestCase):
+    @staticmethod
+    @override
+    def is_supported(config: skips.TestConfig) -> bool:
+        if config.client_lang in _Lang.CPP | _Lang.PYTHON:
```

Comment:
This stretched by brain for a minute. TIL that `enum.Flag` supports both `__contains__` and `__or__`. Cool.

Reply:
Yea, pretty neat trick. It's pretty-much bit flags under the hood.